docs/app/data/footer-navigation.json

```diff
@@ -9,7 +9,8 @@
             "link": "/changelog"
         }]
     }],
-    "logo": "assets/img/UX-Aspects-Logo.png",
+    "logo": "assets/img/UX-graphic-small.png",
```

Comment:
Perhaps change this to:

```json
"logo": {
    "image": "assets/img/UX-graphic-small.png",
    "title": "UX Aspects"
}
```

This way the two logo properties are grouped rather than naming one logo and one logoTitle. It also means in future if we were to, for example, make it go to a link when clicked we could just add a link property under logo rather than creating a new `logoLink` property,